### DISCUSSION TITLE
제 M365 개발자 계정 이름이 LNU FNU 라고 자동으로 붙었는데, 어디서 바꿀 수 있나요?

### DISCUSSION THREAD

**Discussion by justinyoo** [fusiondevkr/pl900#27] [0 공지사항]
M365 개발자 계정을 만들고 나면, 자동으로 LNU FNU 혹은 FNU LNU 라고 이름이 붙습니다. FNU 는 _First Name Unknown_ 의 약자이고, LNU 는 _Last Name Unknown_ 의 약자인데요, 이걸 내가 원하는 이름으로 바꾸고 싶다면 아래와 같이 해 보세요.

1. https://admin.microsoft.com/#/users 페이지로 접속합니다.
2. 만약 로그인을 해야 하면, 내 LNU FNU 이름이 달려 있는 M365 개발자 계정으로 로그인합니다. 그러면 아래와 같은 화면이 보입니다. 이름을 클릭해 보세요.
    ![image](https://user-images.githubusercontent.com/1538528/126528212-72a9a317-f1b7-48c8-957e-82eec3e51466.png)
    &nbsp;
3. 그리고 아래 그림과 같이 **"연락처 정보 관리"** 링크를 클릭해서 이름을 변경하세요.
    ![image](https://user-images.githubusercontent.com/1538528/126528405-9c349f8f-31b5-4b8d-84c2-47b064fc36c6.png)
    &nbsp;
4. 아래와 같이 이름을 변경하고 **"변경 내용 저장"** 버튼을 클릭하면 됩니다.
    ![image](https://user-images.githubusercontent.com/1538528/126528634-c92a432f-e9d8-479a-98eb-a5212ce6fcc3.png)
    &nbsp;
5. 이제 로그아웃하고 다시 로그인해서 이름을 확인해 보세요. 굳이 로그아웃을 하지 않아도 되지만, 한 번 해 주는 것을 권장합니다.

그럼 이제 이름이 바뀐 걸 확인하셨죠?

**Comment by sesme100** [comment]
유용한 정보 감사합니다!!!
